frontend/src/metabase/core/components/AccordionList/AccordionListCell.styled.tsx

```diff
@@ -6,12 +6,16 @@ export interface ListCellItemProps {
   isClickable: boolean;
 }
 
-export const ListCellHeader = styled.div<{ borderBottom?: boolean }>`
+export const ListCellHeader = styled.div<{
+  borderTop?: boolean;
+  borderBottom?: boolean;
+}>`
   border: none;
-  border-top: 1px solid ${color("bg-medium")};
 
-  border-bottom: ${props => (props.borderBottom ? 1 : 0)}px solid
-    ${color("bg-medium")};
+  border: 0px solid ${color("bg-medium")};
+
+  border-bottom-width: ${props => (props.borderBottom ? 1 : 0)}px;
```

Comment:
looks weird tbh, why not to override `border-bottom` instead of those two props?

Reply:
They need to be overridden independently for each other. I'm not super happy with it either but it's the only way I've found doing this whilst also keeping the `AccordionList` fairly backwards compatible.